TAKAware/Screens/Sheets/AnnotationDetailView.swift

```diff
@@ -88,7 +101,13 @@ struct AnnotationDetailReadOnly: View {
                     VStack {
                         Group {
                             Text(annotation!.title ?? "")
-                            Text(annotation!.remarks ?? "")
+                            // TODO: Be smarter about when to show HTML (check for CDATA)
+                            if annotation!.isKML {
+                                HTMLView(htmlString: annotation!.remarks!)
+                                    .frame(height: 200)
```

Comment:
Can we be smarter about the frame height here?

Reply:
We can be, but it's a little trickier. For now the stub code is in to dynamically calculate the height, but we're just fixing to a 100 size height